robot/board.py

```diff
@@ -33,23 +32,28 @@ def _greeting_response(self, data):
         """
         pass
 
-    def _connect(self):
+    def _get_socket(self):
```

Comment:
Could we keep the word "connect" in the name of this function, since that's the most important thing it's doing?

Reply:
I think I was going to have it _not_ connect at one point (just prime the socket), you're right that it's not great as it stands.
I do want it to be clear this is only part of the formation of a connection though. Maybe `_get_connect_socket`? (feels a bit verbose though).

Reply:
I think something like `_get_connected_socket` would be fine. This is very much a "write-once, read-often" piece of code, so I'd value a descriptive-but-accurate name over a brief one.